lib/webhook-proxy.js

```diff
@@ -1,14 +1,38 @@
 module.exports = ({url, port, path, logger}) => {
+  
   try {
-    const SmeeClient = require('smee-client')
-
-    const smee = new SmeeClient({
+    var goodUrl = true; //url is considered to be true;
+    const SmeeClient = require('smee-client');
+    const valid= ValidURL(url);
+    if(!valid)
+    {
+      goodUrl = false; //url is invalid
+      throw "check your Webhook proxy URL";
```

Comment:
Can we change this to something more descriptive:

> The provided Webhook Proxy URL is invalid.

Reply:
Yes, sure. :)